cmd/k8s-keystone-auth/main.go

```diff
@@ -29,32 +28,12 @@ import (
 )
 
 func main() {
-	// Glog requires this otherwise it complains.
-	err := flag.CommandLine.Parse(nil)
-	if err != nil {
-		klog.Fatalf("Unable to parse flags: %v", err)
-	}
-
 	var showVersion bool
 	pflag.BoolVar(&showVersion, "version", false, "Show current version and exit")
 
-	// This is a temporary hack to enable proper logging until upstream dependencies
-	// are migrated to fully utilize klog instead of glog.
-	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
-	klog.InitFlags(klogFlags)
-
 	logs.AddFlags(pflag.CommandLine)
 	keystone.AddExtraFlags(pflag.CommandLine)
 
-	// Sync the glog and klog flags.
-	flag.CommandLine.VisitAll(func(f1 *flag.Flag) {
-		f2 := klogFlags.Lookup(f1.Name)
-		if f2 != nil {
-			value := f1.Value.String()
-			_ = f2.Value.Set(value)
-		}
-	})
```

Comment:
Oh it's used alright: the `kflag.InitFlags` call below [adds `flag.CommandLine` as a flagset](https://github.com/kubernetes/component-base/blob/v0.28.1/cli/flag/flags.go#L51-L59). However, it's not doing anything useful since no upstream dependency should be using glog any more, which means there's nothing valuable to copy. You can test this by sticking some debug printf calls in here (which is what I did :sweat_smile: